app/javascript/components/home/HomeMission.js

```diff
@@ -36,13 +41,15 @@ class HomeMission extends PureComponent {
                             Aplicamos a nossa preferência pelo espírito crítico ao nosso próprio trabalho, e recebemos de bom grado sugestões e críticas para melhorar.
                         </li>
                     </ol>
+                    </div>
+                    <div className="home-our-mission-img">
+                        <img src={infographic} />
+                    </div>
                 </div>
                 <div className="home-voting-countdown">
-                    <div>Counter here</div>
-                    <p>
-                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sit amet efficitur ipsum. Sed dapibus nibh eu ex pretium scelerisque.
-                    </p>
-                    <Button type="primary">Como Votar?</Button>
+                    <img src={voting} />
+                    <Countdown date="2019-10-06T09:00:00" renderer={CountdownRenderer}/>
+                    <Button>Como Votar?</Button>
```

Comment:
We need to add some margin / space before this button

Reply:
I'll keep as it is and wait for UI/UX feedback